Containers/nextcloud/Dockerfile

```
@@ -131,7 +131,7 @@ RUN set -ex; \
     \
     { \
         echo 'session.save_handler = redis'; \
-        echo 'session.save_path = "tcp://${REDIS_HOST}:6379?auth=${REDIS_HOST_PASSWORD}"'; \
+        echo 'session.save_path = "tcp://${REDIS_HOST}/0:6379?auth=${REDIS_HOST_PASSWORD}"'; \
```

It doesn't respect the format of a URI, if it's wanted it's strange...
```suggestion
        echo 'session.save_path = "tcp://${REDIS_HOST}:6379/0?auth=${REDIS_HOST_PASSWORD}"'; \
```

---

Yes, you are right. Made a typo here...